frontend/components.js

```diff
@@ -2,6 +2,7 @@ import {h, Component} from 'preact'
 import {connect} from 'unistore/preact'
 import Cardano from '../wallet/cardano-wallet'
 import actions from './actions'
+import {get as __} from './translations'
```

Comment:
when you export just the object with functions you can just import translations from './tranlsations'

sidenote: you should have a good reason to import something as '_' and you should never have reason to import it as '__'